### PR TITLE
Add new html file with name adderjsAppendChild

### LINKED ISSUE #3: Implement the same task without innerHTML and insertadjacenthtml
To help you can use 
`document.createElement` and `someElement.appendChild()` methods

### PR DESCRIPTION
**Pull request by tsugera**
This solution adds the one written with appendChild ().

### CODE REVIEW

File: adderjsAppendChild.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <ul id="resultBlock"></ul>
+    <script>
+      
+      function giver (){
+        //забираем введённые значения
+        value = document.getElementById("mainInput").value;
+        //создаем li элемент в документе
+        let newElementLi = document.createElement('li');
+        //присваиваем этому новому li элементу значения полученные из текстового поля
+        newElementLi.innerHTML +=  value;
```
Comment (DimonMobile) [inline adderjsAppendChild.html]: Please don't use innerHTML as written in issue description

File: adderjsAppendChild.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            <input type="text" id="mainInput" size="30" placeholder="Напиши сюда, а я ебану в список">
+            <button id="add">Жать</button>
+
+    </div>
+    <ul id="resultBlock"></ul>
+    <script>
+      
+      function giver (){
+        //забираем введённые значения
+        value = document.getElementById("mainInput").value;
```
Comment (DimonMobile) [inline adderjsAppendChild.html]: Value variable is not declared. It's would be better if you will declare it.

**Review by DimonMobile — CHANGES_REQUESTED** [review]

File: adderjsAppendChild.html (hunk 1 above)
Comment (tsugera) [reply adderjsAppendChild.html]: Why?

Comment (DimonMobile) [reply adderjsAppendChild.html]: > Implement the same task without innerHTML and insertadjacenthtml

File: adderjsAppendChild.html [hunk 3]
```diff
@@ -13,23 +13,20 @@
...
-        //создаем li элемент в документе
+        let value = document.getElementById("mainInput").value;
         let newElementLi = document.createElement('li');
-        //присваиваем этому новому li элементу значения полученные из текстового поля
-        newElementLi.innerHTML +=  value;
-        //скармливаем в список с id="resultBlock" полученные значения
+        newElementLi.innerText = value;
         resultBlock.appendChild(newElementLi);
-
+
```
Comment (DimonMobile) [inline adderjsAppendChild.html]: why are there so many empty lines?

**Review by DimonMobile — CHANGES_REQUESTED** [review]

File: adderjsAppendChild.html [hunk 4]
```diff
@@ -6,31 +6,21 @@
...
-    
             <input type="text" id="mainInput" size="30" placeholder="Напиши сюда">
             <button id="add">Жать</button>
-
     </div>
     <ul id="resultBlock"></ul>
-    <script>
-      
-      function giver (){
+    <script><
```
Comment (DimonMobile) [inline adderjsAppendChild.html]: What is that

**Review by DimonMobile — APPROVED** [review]

**Pull request merged**